### PR TITLE
Add app repository

### PR DESCRIPTION
**Pull request by byliuyang**
### Description
Add app table to represents applications managed by third party developer.

### CODE REVIEW

File: backend/app/usecase/repository/app.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package repository
+
+import "github.com/short-d/short/backend/app/entity"
+
+type App interface {
+	FindAppByID(appID string) (entity.App, error)
```
Comment (arberiii) [inline backend/app/usecase/repository/app.go:7]:
```suggestion
	GetAppByID(id string) (entity.App, error)
```

File: backend/app/usecase/repository/app_fake.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+var _ App = (*AppFake)(nil)
+
+// AppFake represents in memory implementation of App repository
+type AppFake struct {
+	apps []entity.App
+}
+
+// FindAppByID fetches an app with given ID from memory.
+func (a AppFake) FindAppByID(appID string) (entity.App, error) {
```
Comment (arberiii) [inline backend/app/usecase/repository/app_fake.go:17]:
```suggestion
func (a AppFake) FindAppByID(id string) (entity.App, error) {
```

File: backend/app/app.go [hunk 3]
```diff
@@ -85,6 +86,7 @@ func Start(
 		kgsBufferSize,
 		kgsRPCConfig,
 		provider.TokenValidDuration(config.AuthTokenLifetime),
+		provider.FeedbackSlackWebHook(config.FeedbackSlackWebHook),
```
Comment (arberiii) [inline backend/app/app.go:89]: provider.FeedbackSlackWebHook doesn't exist

File: backend/app/adapter/sqldb/app_integration_test.go [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+						assert.NotEqual(t, nil, err)
+						return
+					}
+
+					assert.Equal(t, nil, err)
+					assert.Equal(t, testCase.expectedApp.ID, gotApp.ID)
+					assert.Equal(t, testCase.expectedApp.Name, gotApp.Name)
+
+					timeDiff := testCase.expectedApp.CreatedAt.Sub(gotApp.CreatedAt)
+					assert.Equal(t, time.Duration(0), timeDiff)
```
Comment (arberiii) [inline backend/app/adapter/sqldb/app_integration_test.go]: Can't we compare directly expected createdAt with actual?

File: backend/app/app.go (hunk 3 above)
Comment (byliuyang) [reply backend/app/app.go:89]: Accidentally checked it in.

**Pull request merged**